webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -13,7 +13,7 @@ import {
 } from '../../../common/sessionStorage';
 import API from '../../../API';
 
-const defaultNotificationsPollingInterval = 10000;
+const defaultNotificationsPollingInterval = 500000;
```

Comment:
wrong entry?
for future reference, in your development env just use something like
```
NOTIFICATIONS_POLLING=1000000 foreman start
```